resolwe_bio/descriptors/super_sample.yml

```diff
@@ -0,0 +1,263 @@
+- slug: super_sample
+  name: Super sample annotation
+  description: Sample meta data compliant with GEO and ENCODE databases.
+  version: 0.0.1
+  schema:
+    - name: assay_type
+      label: Assay type
+      type: basic:string
+      choices:
+        - label: RNA-seq
+          value: rna-seq
+        - label: ChIP-seq
+          value: chip-seq
+        - label: ATAC-seq
+          value: atac-seq
+        - label: ChIPmentation
+          value: chipmentation
+    - name: biosample_type
+      label: Biosample type
+      type: basic:string
+      choices:
+        - label: Cell line
+          value: cell_line
+        - label: Tissue
+          value: tissue
+        - label: Primary cell
+          value: primary_cell
+    - name: annotator
+      label: Annotator
+      type: basic:string
+    - name: chip_seq
+      label: ChIP-seq
+      group:
+        - name: target_of_assay
+          label: Target of assay
+          type: basic:string
+          choices:
+            - label: Transcription factor
+              value: transcription_factor
+            - label: Histone
+              value: histone
+            - label: Control
+              value: control
+            - label: Chromatin associated protein
+              value: chromatin_associated_protein
+            - label: RNA binding protein
+              value: rna_binding_protein
+            - label: Chromatin remodeller
+              value: chromatin_remodeller
+            - label: Nucleotide modification
+              value: nucleotide_modification
+        - name: antibody_manufacturer
+          label: Antibody manufacturer
+          type: basic:string
+          choices:
+            - label: Abcam
+              value: abcam
+            - label: BioRad
+              value: biorad
+            - label: ThermoFisher
+              value: thermofisher
+            - label: CellSignal
+              value: cellsignal
+            - label: Diagenode
+              value: diagenode
+            - label: ActiveMotif
+              value: activemotif
+        - name: antibody_target
+          label: Antibody target
+          type: basic:string
+        - name: antibody_catalog_number
+          label: Antibody catalog number
+          type: basic:string
+        - name: antibody_lot_number
+          label: Antibody lot number
+          type: basic:string
+    - name: organism
```

Comment:
Consider changing this to `species`. 

Too restrictive list without the `allow_custom_choice` option. The format of the `value` fields as envisioned will break everything that relies on the species name to be in binomial nomenclature. This affects gene knowlege-base, sample basket, existing data objects that will not match the sample basket restrictions (species field on expressions being `Homo sapiens` will not match `homo_sapiens` information on the samples.